scripts/shared.js

```diff
@@ -28,6 +28,15 @@ module.exports = {
       output: path.resolve(__dirname, '../', output)
     };
 
+    // TODO [ToDr] Disabling augmentation and popup in production
+    if (isProd) {
+      manifestOptions.preProcess = (manifest) => {
+        manifest.content_scripts.pop();
```

Comment:
Why not adding a key to each content_script entry of the Manifest instead of nuking the last element without checking if it's the right one ? Or use a regex against the `js` field of each content_script entry that would delete those with files in the `content` directory ? Just doesn't look very safe to just `pop` the last one